### ISSUE TITLE
add DB

### ISSUE THREAD

**Issue by nixoncode** [0xCAF3D0OD/java-docker-environment#1]
can you also add a database as a dependency, like mysql or Postgres

**Comment by 0xCAF3D0OD** [comment]
Yes, it's in progress, but first I'm going to make a tutorial of the process I've followed so far.
